packages/ng-schematics/src/ng-new/index.ts

```diff
@@ -30,6 +30,11 @@ export function newProject(options: OptionsSchema): Rule {
 
 		// TODO:
 		const defaultProjName = "IG Project";
+		let allOptionsProvided: boolean = false;
+
+		if (options.name && options.template && options.theme) {
```

Comment:
@damyanpetev @dafo 
Maybe we should move this under the `name` prompt?
Currently, if you call the schematic like so:
```
ng new --collection=igniteui/angular-schematics --theme=Custom --template=side-nav
```
It will prompt you for a name and then go into the action loop.

Reply:
That's actually better handled than what you'd get with the stand-alone cli equivalent. But technically if you don't provide the name - you ignored the command usage requirements that start the first thing you pass is an argument with name.
I'd accept the partial prompts for just the missing options as an enhancement though, just not for this issue.